main/lsp/requests/workspace_symbols.cc

```diff
@@ -38,22 +183,9 @@ unique_ptr<ResponseMessage> LSPLoop::handleWorkspaceSymbols(LSPTypechecker &type
     }
 
     prodCategoryCounterInc("lsp.messages.processed", "workspace.symbols");
-
-    vector<unique_ptr<SymbolInformation>> result;
-    string_view searchString = params.query;
-    ShowOperation op(*config, "WorkspaceSymbols", fmt::format("Searching for symbol `{}`...", searchString));
-
-    const core::GlobalState &gs = typechecker.state();
-    for (u4 idx = 1; idx < gs.symbolsUsed(); idx++) {
-        core::SymbolRef ref(gs, idx);
-        if (hasSimilarName(gs, ref.data(gs)->name, searchString)) {
-            auto data = symbolRef2SymbolInformation(*config, gs, ref);
-            if (data) {
-                result.push_back(move(data));
-            }
-        }
-    }
-    response->result = move(result);
+    ShowOperation op(*config, "WorkspaceSymbols", fmt::format("Searching for symbol `{}`...", params.query));
```

Comment:
This operation is sufficiently fast now that you can remove this line. This line is what creates the "processing" animation in VS Code. I added it when it seemed like workspace/symbols was very slow (when the problem was it was sending every symbol in p-s to VS Code).